packages/api/src/core/prepareParams.ts

```diff
@@ -192,45 +185,42 @@ export default async function prepareParams(operation: Operation, body?: unknown
     } else if (typeof metadata === 'undefined') {
       // No metadata was explicitly provided so we need to analyze the body to determine if it's a
       // body or should be actually be treated as metadata.
-      if (!hasDigestedParams) {
```

Comment:
Because `Accept` isn't defined as a traditional parameter in an OAS, it'll never show up in the `digestedParameters` object, and will never be here so if you sent an API request of **only** an `Accept` header, it would be treated as a body payload and sent that way.